### PR TITLE
ci: rotate GCP token for CI

### PR DESCRIPTION
**Pull request by josephperrott**

### CODE REVIEW

**Comment by devversion** [comment]
Good idea. I think tests fail because the Circle images do not come with the browser-required dependencies. We could just change to the browsers docker image now that testsnrun in the Bazel host

**Comment by josephperrott** [comment]
@devversion it ended up being easier to rotate the key rather than deal with how the browsers come into the environment.

PTAL

**Review by devversion — DISMISSED** [review]
LGTM, assuming it turns green 😄

File: integration/side-effects/package.json [hunk 1]
```diff
@@ -7,13 +7,13 @@
...
+    "@angular/forms": "file:/usr/local/google/home/jperrott/.cache/bazel/_bazel_jperrott/41b4d1f37a4e02e696982146b10a94be/execroot/angular/bazel-out/k8-fastbuild/bin/integration/side-effects_test.debug.sh.runfiles/angular/packages/forms/npm_package_archive.tar.gz",
+    "@angular/platform-browser": "file:/usr/local/google/home/jperrott/.cache/bazel/_bazel_jperrott/41b4d1f37a4e02e696982146b10a94be/execroot/angular/bazel-out/k8-fastbuild/bin/integration/side-effects_test.debug.sh.runfiles/angular/packages/platform-browser/npm_package_archive.tar.gz",
+    "@angular/router": "file:/usr/local/google/home/jperrott/.cache/bazel/_bazel_jperrott/41b4d1f37a4e02e696982146b10a94be/execroot/angular/bazel-out/k8-fastbuild/bin/integration/side-effects_test.debug.sh.runfiles/angular/packages/router/npm_package_archive.tar.gz",
+    "check-side-effects": "file:/usr/local/google/home/jperrott/.cache/bazel/_bazel_jperrott/41b4d1f37a4e02e696982146b10a94be/execroot/angular/bazel-out/k8-fastbuild/bin/integration/side-effects_test.debug.sh.runfiles/npm/check-side-effects_archive.tar.gz"
```
Comment (devversion) [inline integration/side-effects/package.json:17]: there are a few such files. They are overridden anyway- so that's why CI is passing but FYI

**Review by devversion — DISMISSED** [review]

File: .circleci/config.yml [hunk 2]
```diff
@@ -855,32 +855,35 @@ workflows:
       - build-ivy-npm-packages:
           requires:
             - setup
-      - legacy-unit-tests-saucelabs:
-          requires:
-            - setup
-      - test_aio:
-          requires:
-            - setup
-      - deploy_aio:
```
Comment (devversion) [inline .circleci/config.yml:864]: Actually we do need `deploy_aio` for the image directive right?

Comment (josephperrott) [reply .circleci/config.yml:864]: Do we? Are we needing to push aio again?

Comment (devversion) [reply .circleci/config.yml:864]: yeah, there will be docs for the new directive that is a "feature" right?

**Comment by dylhunn** [comment]
@josephperrott I get the following failure:

```
InvalidTargetBranchError {
  failureMessage: 'Not using last-minor branch for v12 LTS version. PR should be updated to target: 12.2.x'
}
```

Would you be able to merge this for me, if it requires special tooling attention? Thanks in advance!

**Pull request merged**